www/main/common/constants/api-urls.js

```diff
@@ -2,7 +2,7 @@ import { version as appVersion } from '../../../../package.json';
 
 const useStageAPI = appVersion.includes('alpha') || process.env.NODE_ENV === 'development';
 export const API_ENDPOINT = useStageAPI
-  ? 'https://stgapi.sikhitothemax.org'
+  ? 'https://api.sikhitothemax.org'
```

Comment:
This condition appears to be meaningless, as it is referring to the same API.

Reply:
Fixed!